features/main/composite.feature

```diff
@@ -120,6 +120,11 @@ Feature: Retrieve data with Composite identifiers
     }
     """
 
+  Scenario: Get the first composite relation with a missing identifier
+    Given there are Composite identifier objects
+    When I send a "GET" request to "/composite_relations/compositeLabel=1;"
+    Then the response status code should be 400
```

Comment:
> 10.4.5 404 Not Found
   The server has not found anything matching the Request-URI. 

This is by definition what happened here no? Though, one could argue that the url is malformed (hence 400).